doc_src/auto_examples/plot_fit_fake_data.rst

```diff
@@ -0,0 +1,113 @@
+.. only:: html
+
+    .. note::
+        :class: sphx-glr-download-link-note
+
+        Click :ref:`here <sphx_glr_download_auto_examples_plot_fit_fake_data.py>`     to download the full example code
+    .. rst-class:: sphx-glr-example-title
+
+    .. _sphx_glr_auto_examples_plot_fit_fake_data.py:
+
+
+Example for fitdata function
+============================
+This example creates fake data with noise
+then fits the exponential with the fitdata
+function.
+
+
```

Comment:
did you create this, or was it auto-generated?

Reply:
made it